tests/linalg_test.py

```diff
@@ -1619,6 +1619,39 @@ def testToeplitzConstructionWithKnownCases(self):
       [2, 1, 4, 5],
       [3, 2, 1, 4]], dtype=np.float32))
 
+  @jtu.sample_product(
+    size=[1, 2, 3, 10, 100],
```

Comment:
Should also test `size=None`

Reply:
ah here `size` does not correspond to the `size` argument of the `null_space` function. I was just reusing the test from scipy originally, and didnt correct this discrepancy, I will now.